src/jaybenne/jaybenne.cpp

```diff
@@ -573,6 +575,8 @@ TaskStatus UpdateFluid(MeshData<Real> *md) {
         ee += delta;
       });
 
+  Kokkos::Profiling::popRegion(); // End of Jaybenne timestep
```

Comment:
@fglines-nv Am I correct that we don't need a preceding `Kokkos::fence()` here because when using a profiling library like nvtx connector, `popRegion` will include its own `fence()` call?

Reply:
We don't need a `Kokkos::fence()`, that's correct. There won't be any `fence()` or other host-device synchronization points with the `nvtx` connector. `nsys` will capture the timeline of execution on both the CPU and GPU, which will be disconnected.